boards/b-l475e-iot01a/include/periph_conf.h

```diff
@@ -158,6 +164,44 @@ static const spi_conf_t spi_config[] = {
         .tx_dma_chan = 1,
         .rx_dma   = 0,
         .rx_dma_chan = 1,
+#endif
+    },
+    {
+        .dev      = SPI2,
+        .mosi_pin = GPIO_PIN(PORT_D, 4),
+        .miso_pin = GPIO_PIN(PORT_D, 3),
+        .sclk_pin = GPIO_PIN(PORT_D, 1),
+        .cs_pin   = SPI_CS_UNDEF,
```

Comment:
According to the board datasheet, the CS pin is connected to PD5.

Reply:
Hmm but it's not documented as such in the CPU datasheet. Better leave undefined and let the user configure it manually.